### PR TITLE
Adds a new  prop to  that delays loading analytics scripts until the …

### PR DESCRIPTION
**Pull request by markbrocato**
…app is fully interactive.  This helps ensure the best TTI and user experience.

### CODE REVIEW

File: packages/react-storefront/src/analytics.js [hunk 1]
```diff
@@ -5,6 +5,12 @@
 
 let _targets = []
 
+// When false, any fired events will be put in a queue until activate() is called
+let activated = false
+
+// events accumulate here when activated is false
+const queue = []
```
Comment (dijs) [inline packages/react-storefront/src/analytics.js]: Can you add a test for this queue?

File: packages/react-storefront/src/Server.js [hunk 2]
```diff
@@ -161,17 +161,16 @@ export default class Server {
       })
 
       const helmet = Helmet.renderStatic()
+      const chunks = flushChunkNames(stats)
 
       const scripts = [
-        ...getScripts({ stats, chunk: 'bootstrap' }),
```
Comment (dijs) [inline packages/react-storefront/src/Server.js]: Why no bootstrap?

Comment (markbrocato) [reply packages/react-storefront/src/Server.js]: It was removed a while ago actually.  I think by the upgrade to webpack 4.

Comment (dijs) [reply packages/react-storefront/src/Server.js]: Excellent

**Pull request merged**